packages/common/src/search/_internal/hubSearchItemsHelpers/getOgcCollectionUrl.ts

```diff
@@ -13,13 +13,15 @@ import { IApiDefinition } from "../../types/types";
  * @param options request options, including the base OGC api url
  * @returns the collection url
  */
-export function getOgcCollectionUrl(query: IQuery, options: IHubSearchOptions) {
+export function getOgcCollectionUrl(
+  query: IQuery,
+  options: IHubSearchOptions
+): string {
   const apiDefinition = options.api as IApiDefinition;
   // Discussion posts as a target entity will be searchable with one collection,
   // so simply use that for the URL
   if (query.targetEntity === "discussionPost") {
     return `${apiDefinition.url}/collections/discussion-post`;
   }
-  const collectionId = query.collection || "all";
-  return `${apiDefinition.url}/collections/${collectionId}`;
+  return `${apiDefinition.url}/collections/all`;
```

Comment:
This has been a long time coming. We're no longer going to rely on the individual collections that are exposed on by the OGC api. Instead, we'll always hit the "all" collection and serialize our entire query into `?filter`.